tools/hangar/src/__snapshots__/invalid.test.ts.snap

```diff
@@ -174,12 +185,6 @@ exports[`try_no_catch_or_finally.w > wing test (--target sim) - invalid > stderr
 Error at ../../../../examples/tests/invalid/try_no_catch_or_finally.w:1:1 | Missing \`catch\` or \`finally\` blocks for this try statement"
 `;
 
-exports[`types_string_interpolation.w > wing test (--target sim) - invalid > stderr 1`] = `
```

Comment:
Is it intentional that we lost these tests?

Reply:
Yes - see [my comment](https://github.com/winglang/wing/pull/1524#discussion_r1119953551).